### DISCUSSION TITLE
Does this plugin supports the reading of languages from private repos at organizations?

### DISCUSSION THREAD

**Discussion by CxrlosKenobi** [lowlighter/metrics#1605] [Q&A]
I wonder if the code for the languages percent that's rendered in the metrics, reads the private repositories from organizations. Since I've seen that it only reads the code from private repositories at my profile.

**Comment by chasegalloway** [comment]
I would not absolutely take my word for it, as I am not 100% certain, but I was looking at my metrics and noticed that CSS, which is only used within a private repo in my org, was not appearing on the metrics chart, which makes me think that it does not read private repos in organizations by default. However, I think that you can use the plugin_languages_indepth parameter to include repositories that are not under your username
